fabtests/benchmarks/benchmark_shared.c

```diff
@@ -460,6 +485,13 @@ int bandwidth_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
 							tx_buf + offset,
 							opts.transfer_size,
 							remote);
+				} else if (opts.use_fi_more) {
+					flags = set_fi_more_flag(i, j, flags);
+					flags |= FI_REMOTE_CQ_DATA;
```

Comment:
For safety,  better to just do `flags | FI_REMOTE_CQ_DATA` in `ft_post_rma_writemsg` instead of `|=` the `flags` globally.

Reply:
Hmmm nvm, I think this may not be better than what your have now.